lib/stream_data.ex

```diff
@@ -1276,31 +1272,6 @@ defmodule StreamData do
       Enum.at(StreamData.resize(data, 10), 0)
       #=> [[], '\t', '\a', [1, 2], -3, [-7, [10]]]
 
-  Another common example is a tree, where the data exists only in the inner nodes.
```

Comment:
Apparently you dislike my example :-)

Reply:
It's just the same example as above so repeating two trivial "tree" examples seemed very redundant :) I tried taking the improvements to the documentation you made and use them in the descriptive parts.